### PR TITLE
i18n(es): Updated `static-redirect-not-available.mdx` & `redirect-with-no-location.mdx`

### PR DESCRIPTION
**Pull request by dreyfus92**
#### What kind of changes does this PR include?

- New or updated content
- Translated content

#### Description

- What does this PR change? Give us a brief description.
Updated `static-redirect-not-available.mdx` & translated this error `redirect-with-no-location.mdx` based on this commit 05eac779ae7f433955fd1d8909273eb32ac6f479

### CODE REVIEW

File: src/content/docs/es/reference/errors/redirect-with-no-location.mdx [hunk 1]
```diff
@@ -0,0 +1,15 @@
+---
+title: A redirect must be given a location with the Location header.
+i18nReady: true
+githubURL: https://github.com/withastro/astro/blob/main/packages/astro/src/core/errors/errors-data.ts
+---
+
+
+## ¿Qué salió mal?
+A redirect must be given a location with the `Location` header.
```
Comment (Waxer59) [inline src/content/docs/es/reference/errors/redirect-with-no-location.mdx:9]: tienes que borrar esta línea

**Review by Waxer59 — COMMENTED** [review]
Buena PR 🤠, te dejo una sugerencia

**Comment by dreyfus92** [comment]
Gracias por la revisión @Waxer59 😁

**Review by Waxer59 — COMMENTED** [review]
LGTM ✔️

**Review by Waxer59 — APPROVED** [review]
LGTM ✔️

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Thanks for the great work

**Pull request merged**